contracts/disputes/DisputeManager.sol

```diff
@@ -656,7 +694,8 @@ contract DisputeManager is DisputeManagerV1Storage, GraphUpgradeable, IDisputeMa
     function _resolveDisputeInConflict(Dispute memory _dispute) private returns (bool) {
```

Comment:
Since this is now only used for Draws, it'd be good to rename the function to clarify it, e.g. `_resolveDrawnDisputeInConflict` or `_drawDisputeInConflict`.